Apps/US/DynamicsGPMigrationVendor1099/app/src/Upgrade/GPVendor1099MigInstall.Codeunit.al

```diff
@@ -0,0 +1,74 @@
+codeunit 42000 "GP Vendor 1099 Mig. Install"
```

Comment:
Instead of using the OnInstall trigger, could we subscribe to cloud migration enabled event?
If it is possible to change these values, it would be good to get a Reset action thus we can set it back, otherwise we should populate it when we populate the table mappings and other setup records.

Reply:
The user cannot change these values and would only need to be set on install and upgrade.